### PR TITLE
Create docker image for releases

### PR DESCRIPTION
**Pull request by AminArria**
Created Dockerfile to generate docker image and modified `prod.config` to get some values from env:

- `RELX_REPLACE_OS_VARS=true`
- `SSL_CERTFILE=</dir/of/ssl/fullchain.pem>`
- `SSL_KEYFILE=</dir/of/ssl/privkey.pem>`
- `TOKEN_SECRET=<secret_token_hash>`
- `POSTGRES_HOST=<holiday_ping_postgres_1>`
- `POSTGRES_USER=<postgres user>`
- `POSTGRES_PASSWORD=<postgres user password>`
- `POSTGRES_DB=<postgres database>`

### CODE REVIEW

File: conf/prod.config [hunk 1]
```diff
@@ -2,21 +2,22 @@
...
-                 {keyfile, "/etc/letsencrypt/live/holidayping.lambdaclass.com/privkey.pem"},
-                 {token_secret, <<"FIXME!!!">>},
+                 {certfile, "${SSL_CERTFILE}"},
+                 {keyfile, "${SSL_KEYFILE}"},
+                 {token_secret, "${TOKEN_SECRET}"},
                  {token_expiration, 86400},
                  {verification_code_expiration, 300},
                  {password_code_expiration, 300},
                  {checker_interval, 900000},
-                 {email_enabled, true},
```
Comment (facundoolano) [inline conf/prod.config]: isn't this necessary for production?

**Comment by facundoolano** [comment]
Did you test the github login flow?

File: conf/prod.config (hunk 1 above)
Comment (AminArria) [reply conf/prod.config]: Yep, my bad. Fixed in 33aacce

**Comment by facundoolano** [comment]
I think you need to add the github and amazon SES credentials somewhere to get email and github login working

**Comment by AminArria** [comment]
Github and amazon SES credentials are directly looked for as OS env vars. 

I tested the Github login and it works fine.

**Comment by AminArria** [comment]
Travis fails due to not having updated holidays

**Review by facundoolano — APPROVED** [review]

**Pull request merged**